plugins/ipconsole/ipython_view.py

```diff
@@ -402,7 +427,20 @@ def __init__(self):
         Initialize console view.
         '''
         Gtk.TextView.__init__(self)
-        self.modify_font(Pango.FontDescription('Mono'))
+        pango_ctx = self.get_pango_context()
+        chosen = None
+        for f in pango_ctx.list_families():
+            name = f.get_name()
+            # These are known to show e.g U+FFFC
+            if name in ["Courier New", "Courier Mono"]:
+                chosen = name
+                break
+            if name in ["Liberation Sans"]:
```

Comment:
The problem with this is that Liberation Sans is a quite popular font in the Linux world while the other two aren't. Liberation Sans is a proportional font, which means it will cause misalignments on stuff like the `^` indicator in tracebacks:

```pytb
  File "<string>", line 1
    test!
        ^
SyntaxError: invalid syntax
```

I wonder why Accerciser wants to show [U+FFFC](https://www.compart.com/en/unicode/U+FFFC); I've never seen it before, and we definitely don't need it for Exaile. I think it's best to comment out this `if` block.

Reply:
Why don't you use `Liberation Mono` instead of `Liberation Sans`? Afaik, it also displays U+FFFC (`￼`).

Reply:
No, Liberation Mono does not contain that character. You can check it with `fc-list ':charset=fffc' | sort`. As far as I can tell, the most popular font with that character is DejaVu Sans Mono (installation stats: [80% on Debian](https://qa.debian.org/popcon.php?package=fonts-dejavu), [85% on Arch](https://pkgstats.archlinux.de/packages/ttf-dejavu)).

Reply:
Should I just drop the commit that introduces this change (so we always keep using `Mono`)? 
Or would we prefer to keep the commit to remain in sync with upstream, and just comment out the `Liberation Sans` if block?

Reply:
Might be useful keeping it commented, so that in the future we know that we intentionally disable that part of the code.

Reply:
OK, done.